audeer/core/io.py

```diff
@@ -276,45 +276,51 @@ def file_extension(
 
 def list_dir_names(
         path: typing.Union[str, bytes],
+        basename: bool = False,
```

Comment:
I suggest to make `basename` a key-word only argument like in `list_file_names`

Reply:
Good catch, that was of course my plan.

I changed it accordingly.